packages/base/components/Content/ContentRichText.vue

```diff
@@ -0,0 +1,66 @@
+<script setup>
+import { marked } from "marked";
+
+const props = defineProps({
+  text: {
+    type: String,
+    required: true,
+  },
+  // TODO: find a better and cleaner solution
+  // or remove in the future?
+  richTextIsCard: {
+    type: Boolean,
+    default: true,
+  },
+});
+
+const html = computed(() => marked.parse(props.text));
+</script>
+<template>
+  <div
+    data-qa="markdown"
+    class="row"
+    :class="{ 'mb-5': richTextIsCard }"
+    tag="section"
+  >
+    <div class="col col-12" :class="richTextIsCard ? 'col-lg-6' : 'col-lg-9'">
+      <div v-if="html && richTextIsCard" class="card">
```

Comment:
Should the condition on the presence of `html` be moved up higher, e.g. to the root element?